frontend/server.js

```diff
@@ -11,10 +11,15 @@ if (dotenv.parsed) {
   PORT = dotenv.parsed.PORT || PORT;
 }
 
+// maxAge and mustRevalidate control how the client caches application files. The settings
+// below allows the client to cache content, but the client must check to see if the content
+// is stale. Our app serves content with eTags, so this results in a status 304 Not Modified
+// response, unless the content has been updated.
 const app = express();
 app.use(
   cacheControl({
-    noStore: true,
+    mustRevalidate: true,
+    maxAge: 0,
```

Comment:
is `maxAge` needed? It doesn't seem to be doing anything.

Reply:
It should override any default max-age behavior for a browser or proxy server that interprets the response. I would think that some proxy servers are pretty aggressive about what they're trying to cache so this makes it explicit. Max-age: 0 and must-revalidate are commonly used together and I think make the combo makes sense for our app.

Reply:
Thanks for the explanation!